HealthSDK/GiniHealthSDKPinning/README.md

```diff
@@ -13,15 +14,15 @@ By integrating this SDK into your application you can allow your users to easily
 
 ## Documentation
 
-Further documentation with installation, integration or customization guides can be found in our [website](https://developer.gini.net/gini-health-sdk-pinning-ios/docs/).
+Further documentation with installation, integration or customization guides can be found in our [website](https://developer.gini.net/gini-mobile-ios/GiniHealthSDK/index.html).
 
 ## Example apps
 
-We are providing example app for Swift. This app demonstrates how to integrate the Gini Health SDK with the Component API of Gini Capture library. To run the example project, clone the repo and run `pod install` from the Example directory first.
+We are providing example app for Swift. This app demonstrates how to integrate the Gini Health SDK with the Component API of Gini Capture library.
```

Comment:
To avoid dupllication I would simply link to the Example apps section in `HealthSDK/GiniHealthSDK/README.md`: https://github.com/gini/gini-mobile-ios/tree/main/HealthSDK/GiniHealthSDK#example-apps

Reply:
Thank you for the comments 👍🏻 I went through all of them -> waiting until all builds are succeeded.